RobotCode/src/main/java/frc/robot/Autonomous.java

```diff
@@ -93,19 +92,53 @@ private Autonomous(RobotContainer robotContainer)
         logFine("GUI initialized!");
     }
 
-    /** Creates an instance and adds auto options to Shuffleboard */
+    /** Registers commands for building autos through PathPlanner. */
+    private void registerNamedCommands()
+    {
+        logFine("Registering named commands...");
+
+        // Get subsystems
+        final ShooterMountSubsystem ShooterMount = RobotContainer.getShooterMount();
+        final ShooterSubsystem Shooter = RobotContainer.getShooter();
+        final IndexerSubsystem Indexer = RobotContainer.getIndexer();
+        final IntakeSubsystem Intake = RobotContainer.getIntake();
+
+        // Initialize commands
+        final IntakeCommand IntakeCommand = new IntakeCommand(Intake, Indexer, ShooterMount);
+        NamedCommands.registerCommand("Intake", IntakeCommand);
+
+        final ShootCommand ShootCommand = new ShootCommand(Shooter, Indexer);
+        NamedCommands.registerCommand("Shoot", ShootCommand);
+
+        // Populate rotation commands
+        for (double rot : AutoConstants.AutoShooterMountRotations)
```

Comment:
Possibly rename rot to something more descriptive? Doesn't seem like a big deal since it's still pretty clear as to what it does if you read the code, but may need a name change.

Reply:
Added Javadoc comment with details.